core/state_transition.go

```diff
@@ -365,6 +370,15 @@ func (st *StateTransition) preCheck() error {
 			}
 		}
 	}
+	// Check that EIP-7702 authorization list signatures are well formed.
+	for i, auth := range msg.AuthList {
+		switch {
+		case auth.R.BitLen() > 256:
+			return fmt.Errorf("%w: address %v, authorization %d", ErrAuthSignatureVeryHigh, msg.From.Hex(), i)
+		case auth.S.BitLen() > 256:
+			return fmt.Errorf("%w: address %v, authorization %d", ErrAuthSignatureVeryHigh, msg.From.Hex(), i)
```

Comment:
I'm wondering whether we need to do a signature malleability check here? We're doing one in `authorization.Authority()`. So if a signature is malleable, the pre-check would pass, so the transaction would be valid, but the authorization itself would be skipped afaict